### PR TITLE
[Merged by Bors] - Bump api to v1.37.0

### PR DESCRIPTION
**Pull request by poszu**
## Description

- updated api dep
- updated the `v2alpha1.Activation(Stream)Service`

## Test Plan

n/a

## TODO

- [x] Explain motivation or link existing issue(s)
- [x] Test changes and document test plan
- [ ] Update documentation as needed
- [ ] Update [changelog](../CHANGELOG.md) as needed

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,6 +17,12 @@ See [RELEASE](./RELEASE.md) for workflow instructions.
 
 * [#5807](https://github.com/spacemeshos/go-spacemesh/pull/5807) Implement SMIP-0002: remove vesting vault cliff.
 
+* [#5791](https://github.com/spacemeshos/go-spacemesh/pull/5791) Speed up ATX queries.
+  This also fixes ambiguity of nonces for equivocating identities.
+
+* [#5856](https://github.com/spacemeshos/go-spacemesh/pull/5856) Bump github.com/spacemeshos/api/release/go to v1.37.0.
+  Updated v2alpha1.ActivationTx type used in v2alpha1.Activation(Stream)Service.
```
Comment (fasmat) [inline CHANGELOG.md]: I don't think this needs to be included in the changelog - v2alpha1 is still subject to change and we never had it mentioned in the CHANGELOG before I believe

**Review by fasmat — APPROVED** [review]

**Comment by fasmat** [comment]
Semi-related point: I believe we might need to include the commitment ATX in the domain type. It depends a bit how we want to do the validation of an incoming ATX. We could just fetch the latest (or any) valid ATX of the smesher to get the commitment if it is part of the domain type, wdyt?

**Comment by poszu** [comment]
> Semi-related point: I believe we might need to include the commitment ATX in the domain type. It depends a bit how we want to do the validation of an incoming ATX. We could just fetch the latest (or any) valid ATX of the smesher to get the commitment if it is part of the domain type, wdyt?

Yeah, it could speed up handling incoming ATXs. The handler reads the previous ATX from the DB anyway to do some checks against it. Having the commitment ATX always present in the previous ATX would save us another DB query to obtain it.

**Comment by poszu** [comment]
bors merge

**Pull request closed without merging**